service/src/main/java/uk/gov/hmcts/reform/fpl/model/document/SealType.java

```diff
@@ -7,7 +7,7 @@
 @Getter
 public enum SealType {
     ENGLISH("static_data/familycourtseal.png"),
-    BILINGUAL("static_data/familycourtseal-bilingual.png"),
+    WELSH("static_data/familycourtseal-welsh.png"),
```

Comment:
do we want to keep the "fpl-service" copy of these files to be the same names as the ones in rdo-docmosis?